docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -58,6 +58,67 @@ dagster-dbt project scaffold --project-name my-dagster-project --dbt-project-dir
 
 This creates a directory called `my-dagster-project/` inside the current directory. The `my-dagster-project/` directory contains a set of files that define a Dagster project.
 
+</TabItem>
+<TabItem name="Option 2: Use an existing Dagster project">
```

Comment:
```suggestion
<TabItem name="Option 2: Load your dbt project in an existing Dagster project">
```

Reply:
Done in [ce9d1a1](https://github.com/dagster-io/dagster/pull/21817/commits/ce9d1a18fb46e5753d78d93e671b398825a9ec79)